x-pack/legacy/plugins/uptime/public/pages/page_header.tsx

```diff
@@ -4,75 +4,51 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import React, { useEffect } from 'react';
+import { ChromeBreadcrumb } from 'kibana/public';
+import { EuiFlexGroup, EuiFlexItem, EuiTitle, EuiSpacer } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
-import React, { useEffect, useState } from 'react';
-import { useRouteMatch } from 'react-router-dom';
-import { EuiTitle, EuiFlexGroup, EuiFlexItem, EuiSpacer } from '@elastic/eui';
 import { UptimeDatePicker } from '../components/functional/uptime_date_picker';
-import { getMonitorPageBreadcrumb, getOverviewPageBreadcrumbs } from '../breadcrumbs';
-import { stringifyUrlParams } from '../lib/helper/stringify_url_params';
-import { getTitle } from '../lib/helper/get_title';
-import { UMUpdateBreadcrumbs } from '../lib/lib';
-import { useUrlParams } from '../hooks';
-import { MONITOR_ROUTE } from '../../common/constants';
-import { Ping } from '../../common/graphql/types';
+import { useKibana } from '../../../../../../src/plugins/kibana_react/public';
 
 interface PageHeaderProps {
-  monitorStatus?: Ping;
-  setBreadcrumbs: UMUpdateBreadcrumbs;
+  headingText: string;
+  breadcrumbs: ChromeBreadcrumb[];
+  datePicker: boolean;
 }
 
-export const PageHeaderComponent = ({ monitorStatus, setBreadcrumbs }: PageHeaderProps) => {
-  const monitorPage = useRouteMatch({
-    path: MONITOR_ROUTE,
-  });
-
-  const [getUrlParams] = useUrlParams();
-  const { absoluteDateRangeStart, absoluteDateRangeEnd, ...params } = getUrlParams();
-
-  const headingText = !monitorPage
-    ? i18n.translate('xpack.uptime.overviewPage.headerText', {
-        defaultMessage: 'Overview',
-        description: `The text that will be displayed in the app's heading when the Overview page loads.`,
-      })
-    : monitorStatus?.url?.full;
-
-  const [headerText, setHeaderText] = useState(headingText);
+export const BaseBreadcrumb: ChromeBreadcrumb = {
+  text: i18n.translate('xpack.uptime.breadcrumbs.overviewBreadcrumbText', {
+    defaultMessage: 'Uptime',
+  }),
+  href: '#/',
+};
```

Comment:
I just realized, when on monitor page, and if you click on uptime link to go back to overview page, it resets all filters, i think we can persist all those filters from overview page once we navigate to monitor page.

Especially date range filters, can append those here into href?

Reply:
Yes, good call